scipy/stats/_morestats.py

```diff
@@ -321,7 +327,7 @@ def kstat(data, n=2, *, axis=None):
     lambda x: x, result_to_tuple=lambda x: (x,), n_outputs=1, default_axis=None
 )
 def kstatvar(data, n=2, *, axis=None):
-    r"""Return an unbiased estimator of the variance of the k-statistic.
+    r"""Return an unbiased estimator of the variance of the k-statistic [1]_.
 
     See `kstat` for more details of the k-statistic.
```

Comment:
To avoid 
<img width="631" alt="image" src="https://github.com/scipy/scipy/assets/6570539/70a22c7b-d8f1-4cea-8bea-b15107516b0c">

```suggestion
    r"""Return an unbiased estimator of the variance of the k-statistic.

    See `kstat` and [1]_ for more details about the k-statistic.
```
If you think this can be committed with `[skip ci]`, please go for it and I'll merge.